kubernetes/docs/deploy-to-ibmcloud.md

```diff
@@ -0,0 +1,165 @@
+# Deploy to IBM Cloud
+
+## Create a Kubernetes cluster
+
+https://cloud.ibm.com/kubernetes/clusters
+
+![clusters](../imgs/clusters.png)
+
+Click on `shopping-cluster` to view:
+
+![shopping-cluster](../imgs/shopping-cluster.png)
+
+## Set up CLI for IBM Cloud
+
+https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install
+
+## Connect to IBM Cloud with CLI
+
+If this is your first time connecting to an IBM Cloud cluster, see the
+[full setup directions](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install).
+
+![connect-via-cli](../imgs/connect-via-cli.png)
+
+## Set up helm
+
+https://cloud.ibm.com/docs/containers?topic=containers-helm
+
+```sh
+kubectl create serviceaccount tiller -n kube-system
+kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller -n kube-system
+kubectl get serviceaccount -n kube-system tiller
+helm init --service-account tiller
+```
+
+## Set up IBM Cloud container registry
+
+https://cloud.ibm.com/kubernetes/registry/main/start
+
+## Upload images to IBM Cloud container registry
+
+1. Login into IBM Cloud container registry
+
+```sh
+ibmcloud cr login
+```
+
+2. Use an existing namespace for the container registry or create a new one:
+
+Please substitute `<your-namespace>` to container registry namespace you create
+for IBM Cloud.
+
+Existing namespaces can be listed:
+
+```sh
+ibmcloud cr namespaces
+```
+
+You can add a namespace as follows:
+
+```sh
+ibmcloud cr namespace-add <your-namespace>
+```
+
+3. Upload docker images
+
+```sh
+export CR_NAMESPACE=<your-namespace>
+docker tag loopback4-example-shopping:1.0.0 us.icr.io/${CR_NAMESPACE}/loopback4-example-shopping:1.0.0
+docker push us.icr.io/${CR_NAMESPACE}/loopback4-example-shopping:1.0.0
+
+docker tag loopback4-example-recommender:1.0.0 us.icr.io/${CR_NAMESPACE}/loopback4-example-recommender:1.0.0
+docker push us.icr.io/${CR_NAMESPACE}/loopback4-example-recommender:1.0.0
+```
+
+Please note that the fully qualified docker image name is in the form of
+`<registry>/<namespace>/<repository>:<tag>`. For example,
+`us.icr.io/raymondfeng/loopback4-example-shopping:1.0.0` represents the
+following:
+
+- registry - us.icr.io
+- namespace - raymondfeng
+- repository - loopback-example-shopping
+- tag - 1.0.0
+
+## Install the helm chart
+
+1. Update `kubernetes/shopping-app/ibmcloud-values.yaml`:
+
+```yaml
+images:
+  recommender: us.icr.io/<your-namespace>/loopback4-example-recommender:1.0.0
+  shopping: us.icr.io/<your-namespace>/loopback4-example-shopping:1.0.0
+```
+
+2. Install the helm chart to IBM Cloud
+
+```sh
+helm install --name shopping-app -f kubernetes/shopping-app/ibmcloud-values.yaml kubernetes/shopping-app/
```

Comment:
Step still missing

We need:

helm dependency build kubernetes/shopping-app

before 

helm install --name shopping-app -f kubernetes/shopping-app/ibmcloud-values.yaml kubernetes/shopping-app/